src/viking.js

```diff
@@ -1,11 +1,121 @@
-// Soldier
+// Soldier 
+
 function Soldier() {}
 
-// Viking
+function Soldier(healthArg, strengthArg) {
```

Comment:
You are declaring the Soldier function twice, you could remove the first declaration.